src/components/Widgets/SelectInput/SelectInput.jsx

```diff
@@ -0,0 +1,379 @@
+import React from 'react';
```

Comment:
invece di portare dentro questo componente che tra l'altro usa una libreria in piu, hai provato a usare la select di design-react-kit che è stata usata in sortbywidget?

Reply:
sì a me non funziona, da un errore sull'uso degli eventi, anche usandola nel modo più basic possibile, come da documentazione

Reply:
Io questa select l'ho lasciata perchè quella del design-react-kit in realtà ancora non è stata usata, non ho ben presente come funzioni, nel sortBy abbiamo utilizzato un dropdown menu neanche propriamente accessibile, lascerei questa al momento poi vediamo